docs/src/tutorials_template/tut_beam.md

```diff
@@ -0,0 +1,5 @@
+# Example file
```

Comment:
Please delete the original files in the `tutorials` directory and then add this directory to the `.gitignore`.